internal/app/networkd/pkg/reg/reg_test.go

```diff
@@ -28,13 +30,13 @@ type NetworkdSuite struct {
 
 func TestNetworkdSuite(t *testing.T) {
 	// Hide all our state transition messages
-	// log.SetOutput(ioutil.Discard)
+	log.SetOutput(ioutil.Discard)
```

Comment:
this is going to affect all the tests running after this one, so I think better way should be to restore previous output device when test finishes; tests for different packages are launched as separate binaries, but I guess still better hygiene

Reply:
+1